src/libraries/System.Private.Xml/src/System/Xml/Schema/SchemaCollectionCompiler.cs

```diff
@@ -2439,98 +2439,6 @@ private ContentValidator CompileComplexContent(XmlSchemaComplexType complexType)
             }
         }
 
-#if DEBUG
```

Comment:
I'd prefer we left this debug code in here as it is occasionally useful

Reply:
Restored! Thanks for the feedback!